include/z64.h

```diff
@@ -76,7 +76,7 @@ typedef struct {
     /* 0x0 */ s8 segment;
     /* 0x2 */ s16 type;
     /* 0x4 */ void* params;
-} AnimatedTexture; // size = 0x8
+} MaterialAnimation; // size = 0x8
```

Comment:
I think I like the name `AnimatedMaterial` better.
If we want to shorten it `AnimatedMat` is also good.

Reply:
I can do that, but what about all the other places that I use MatAnim and stuff like that, should I change those too? There is a decent chance that I might miss some 🤔